forge/db/views/Project.js

```diff
@@ -210,6 +210,39 @@ module.exports = function (app) {
         return result
     }
 
+    app.addSchema({
+        $id: 'DashboardInstanceSummary',
+        type: 'object',
+        properties: {
+            id: { type: 'string' },
+            name: { type: 'string' },
+            url: { type: 'string' },
+            createdAt: { type: 'string' },
+            updatedAt: { type: 'string' },
+            links: { $ref: 'LinksMeta' },
+            application: { type: 'object', additionalProperties: true },
+            flowLastUpdatedAt: { type: 'string' },
+            status: { type: 'string' },
+            settings: { type: 'object', additionalProperties: true }
```

Comment:
```suggestion
            settings: {
                type: 'object',
                properties: {
                    dashboard2UI: { type: 'string' }
                }
            }
```

To avoid any other data getting though (via regression etc), we should probably be specific here.

Doesnt need to be a new schema at this time, just be explicit about what is included.